Xamarin.Forms.Platform.Android/Renderers/FormsEditText.cs

```diff
@@ -6,11 +6,11 @@
 
 namespace Xamarin.Forms.Platform.Android
 {
-	public class EntryEditText : EditText, IDescendantFocusToggler
+	public class FormsEditText : EditText, IDescendantFocusToggler
```

Comment:
It might break a custom renderer which subclasses `EntryRenderer` or `EditorRenderer` and then explicitly casts `Control` to `EditText` or `EntryEditText`, but that's a pretty low risk and a very simple change if it does happen. I'm happy to take the risk. I'll call out the possible issues in the PR description so they make it into the release notes (and tag @jassmith on this comment to give him the opportunity to veto).